### DISCUSSION TITLE
How show placeholder nativeselect ?

### DISCUSSION THREAD

**Discussion by faruuhan** [mantinedev/mantine#1416] [Q&A] [answered]
why placeholder nativeselect not show?

**Comment by rtivital** [comment, marked as answer]
`select` element does not have a placeholder and it is hard to customize the value, you can set placeholder manually by setting value to empty string and adding styles based on that value – https://codesandbox.io/s/quiet-bush-yzojgd?file=/src/App.tsx

**Reply by pwong3** [reply]
update 5.6.2 has broken this solution, the placeholder no longer shows up

**Reply by rtivital** [reply]
Yes, this feature is no longer supported
